db/hints/internal/hint_endpoint_manager.cc

```diff
@@ -45,54 +45,51 @@ constexpr std::chrono::seconds HINT_FILE_WRITE_TIMEOUT = std::chrono::seconds(2)
 
 } // anonymous namespace
 
-bool hint_endpoint_manager::store_hint(schema_ptr s, lw_shared_ptr<const frozen_mutation> fm, tracing::trace_state_ptr tr_state) noexcept {
+future<> hint_endpoint_manager::do_store_hint(schema_ptr s, lw_shared_ptr<const frozen_mutation> fm, tracing::trace_state_ptr tr_state) {
+    ++_hints_in_progress;
```

Comment:
The diff is quite unreadable due to the indentation change. Please split it into two commits:

- In one commit, move the lambda to a separate function but try to keep the old indentation in order to minimize the diff,
- In the other commit, fix the indentation.

This is what we usually do. Example: https://github.com/scylladb/scylladb/commit/aa378c458e3e10440b01f45dcce74496b7ad7914 + https://github.com/scylladb/scylladb/commit/66292b4baa27d3f7a86f28634be3da8b6a9450a9